src/MoBi.Assets/AppConstants.cs

```diff
@@ -804,6 +804,11 @@ public static string ChangeSimulationConfiguration(string simulationName)
          {
             return $"Changed the simulation configuration of simulation '{simulationName}'";
          }
+
+         public static string SetModuleMergeBehavior(string moduleName, string newMergeBehavior)
+         {
+            return $"Set merge behavior to '{newMergeBehavior}' for module '{moduleName}'";
```

Comment:
Should we have as text in the UI that says what this means? (When you create the module?)

Reply:
ok, but that's the command description. I'll make the combobox where you pick it have an explanation and I'll probably switch the explanation when the combobox changes.

Reply:
What about like this?
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/261477/40f90da2-370f-4126-be68-687147c9465c)